ontology/yaml/resources/SAFETY/entity_types/ABSTRACT.yaml

```diff
@@ -79,18 +79,56 @@ FA2X:
   uses:
   - fire_alarm_1
   - fire_alarm_2
+  opt_uses:
+  - primary_fire_alarm
+  - secondary_fire_alarm
     
 WLS:
   id: "944988050314756096"
   description: "Water leak location monitoring."
   is_abstract: true
   uses:
   - water_leak_cable_distance_sensor
-  
+
 LM:
   id: "15773300929664647168"
   description: "Tank level monitoring."
   is_abstract: true
   uses:
   - level_status 
 
+HDS:
+  description: "Standard Hydrogen sensor."
+  is_abstract: true
+  implements:
+  - EQUIPMENT
+  
+WCR:
+  description: "Water conditioner"
+  is_abstract: true
+  uses:
+  - run_status
+  implements:
+  - EQUIPMENT
+
+WSR:
```

Comment:
Same as above

Reply:
I'll move them both to PLUMBING. Thanks.